event_routing_backends/models.py

```diff
@@ -240,6 +242,12 @@ def _match_event_for_host(self, original_event, host_config):
             bool
         """
         for key, value in host_config.get('match_params', {}).items():
-            if get_value_from_dotted_path(original_event, key) != value:
+            original_event_value = get_value_from_dotted_path(original_event, key)
+            if isinstance(value, list):
+                if original_event_value not in value:
```

Comment:
@rehanedly Please add regex search within the elements of list as well. I am also updating the example in ADR to reflect this.

Reply:
@ayub02 It's done